src/core/ColorPicker/ColorSwatch.test.tsx

```diff
@@ -8,9 +8,7 @@ import { ColorSwatch } from './ColorSwatch';
 import { ColorValue } from '../utils';
 
 it('should render in its most basic state with ColorValue', () => {
-  const { container } = render(
-    <ColorSwatch color={ColorValue.create('#9BA5AF')} />,
-  );
+  const { container } = render(<ColorSwatch color={'#9BA5AF'} />);
```

Comment:
This test name is inaccurate now, and there is no difference between this and the second test.

Reply:
Updated tests